integrations/serverpod_cloud_storage_gcp/lib/src/google_cloud_storage.dart

```diff
@@ -0,0 +1,289 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:crypto/crypto.dart';
+import 'package:hex/hex.dart';
+import 'package:gcloud/storage.dart';
```

Comment:
Suggestion: Reading up on this dependency, it is tagged as experimental by Google. I'm a little reluctant to add it because of that. It is also possible to implement this same functionality with `googleapis/storage`, it is a bigger deviation from the current solution, but I believe it would be more resilient.

Reply:
It does have a status of experimental, and I don't know what their timeline to GA is, or if it will make it to GA... Will look into `googleapis/storage` to see how it works